src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -21,6 +21,52 @@
 #include "ray/gcs/pb_util.h"
 #include "ray/stats/stats.h"
 
+namespace {
+const ray::rpc::ActorDeathCause GenRuntimeEnvFailedCause(const std::string &error_msg) {
+  ray::rpc::ActorDeathCause death_cause;
+  death_cause.mutable_runtime_env_failed_context()->set_error_message(error_msg);
+  return death_cause;
+}
+
+const ray::rpc::ActorDeathCause GenNodeDiedCause() {
+  ray::rpc::ActorDeathCause death_cause;
+  death_cause.mutable_node_died_context()->set_error_message(
+      "The actor is dead because the node where actor resides is dead.");
```

Comment:
Please carefully check the grammar of error messages

Reply:
Will start using grammarly... lol